Core/Executor/UserGroupManager.php

```diff
@@ -31,14 +31,17 @@ protected function create()
 
         $parentGroupId = $this->dsl['parent_group_id'];
         $parentGroupId = $this->referenceResolver->resolveReference($parentGroupId);
-
-        $parentGroup = $userService->loadUserGroup($parentGroupId);
+        $parentGroup = $this->userGroupMatcher->matchOneByKey($parentGroupId);
```

Comment:
Sure, but the matcher can use different formats of Ids, while the User Services expects and id here, no?